apps/app/pages/_app.js

```diff
@@ -13,6 +13,7 @@ import { ThemeProvider } from 'styled-components';
 import theme from '../styles/theme';
 import Script from 'next/script';
 import iubendaScript from '../scripts/iubendaScript';
+import { OnboardingProvider } from '../../website/src/context/OnboardingContext'
```

Comment:
I will double-check to see if this is the right _app.js file to import context api